pepper-apis/src/main/resources/db-changes/seed/DDP-4222-add-reached-ageup-prep-event.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="sampath" id="12182019-insert-ageup-prep-event">
+        <insert tableName="event_trigger_type">
+            <column name="event_trigger_type_code" value="REACHED_AOM_PREP"/>
```

Comment:
@spookymotion please note that @ssettipalli is adding this trigger type ahead of your work on DDP-4222 since it is blocking him.